File: 백준/Bronze/32498. Call for Problems/Call for Problems.java
import java.io.BufferedReader;
import java.io.IOException;
import java.io.InputStreamReader;
public class Main {
    public static void main(String[] args)throws IOException{
        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
        int num = Integer.parseInt(br.readLine());
        int result = 0;
        for(int i = 0; i < num; i++){
            int input = Integer.parseInt(br.readLine());
            if(input%2 != 0){
                result++;
            }
        }
        System.out.println(result);
    }
}

File: 백준/Bronze/32498. Call for Problems/Call for Problems.cs
int num = int.Parse(Console.ReadLine());
int result = 0;
for (int i = 0; i < num; i++)
{
    int input = int.Parse(Console.ReadLine());
    if (input % 2 != 0)
    {
        result++;
    }
}
Console.Write(result);